bazel/cloudbuild.yaml

```diff
@@ -12,10 +12,13 @@ steps:
 - name: 'gcr.io/$PROJECT_ID/bazel'
   args: ['run', '--spawn_strategy=standalone', '//:target', '--verbose_failures']
   dir: 'examples'
+- name: 'gcr.io/$PROJECT_ID/bazel'
+  args: ['run', '--spawn_strategy=standalone', '//:target', '--verbose_failures', '--', 'world']
+  dir: 'examples'
 
 # Example was built as bazel:target.
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['run', 'bazel:target']
+  args: ['run', 'bazel:target', 'world']
```

Comment:
Does this test need to include the `--` separator to test that the script is doing the right thing?

Reply:
Yes, but I also don't care about exercising the argument checking binary via docker. So I've since dropped this step entirely.